From linas@austin.ibm.com Tue Jan 10 13:18:42 2006
From: linas@austin.ibm.com (linas)
Date: Tue, 10 Jan 2006 15:15:47 -0600
To: greg@kroah.com
Cc: <johnrose@us.ibm.com>, Olaf Hering <olh@suse.de>
Subject: [PATCH 1/2] PCI Hotplug/powerpc: module build break
Message-ID: <20060110211547.GM26221@austin.ibm.com>
Content-Disposition: inline


The RPAPHP hoplug driver will not build as a module, because it calls
on pci_claim_resource(), which is not exported. This exports the symbol.
Problem reported by Olaf Hering <olh@suse.de>

A grep indicates that building drivers/parisc/lba_pci.c
would have trouble building as a module for the same reason.

Signed-off-by: Linas Vepstas <linas@austin.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/pci/setup-res.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/pci/setup-res.c
+++ gregkh-2.6/drivers/pci/setup-res.c
@@ -112,6 +112,7 @@ pci_claim_resource(struct pci_dev *dev, 
 
 	return err;
 }
+EXPORT_SYMBOL_GPL(pci_claim_resource);
 
 int pci_assign_resource(struct pci_dev *dev, int resno)
 {
